spaceship/lib/spaceship/connect_api/models/app_clip_version.rb

```diff
@@ -0,0 +1,42 @@
+require_relative '../model'
+require_relative './app_clip_version_localization'
+
+module Spaceship
+  class ConnectAPI
+    class AppClipVersion
+      include Spaceship::ConnectAPI::Model
+
+      module InvocationVerb
+        OPEN = 'OPEN'
+        VIEW = 'VIEW'
+        PLAY = 'PLAY'
+      end
+
+      attr_accessor :invocation_verb
+
+      attr_mapping(
+        'invocationVerb' => 'invocation_verb'
+      )
+
+      def self.type
+        'appClipVersions'
+      end
+
+      def get_app_clip_version_localizations(client: nil, filter: {}, includes: Spaceship::ConnectAPI::AppClipVersionLocalization::ESSENTIAL_INCLUDES, limit: nil, sort: nil)
+        client ||= Spaceship::ConnectAPI
```

Comment:
```suggestion
      def get_app_clip_version_localizations(client: nil, filter: {}, includes: nil, limit: nil, sort: nil)
        client ||= Spaceship::ConnectAPI
        includes ||= Spaceship::ConnectAPI::AppClipVersionLocalization::ESSENTIAL_INCLUDES
```

Reply:
Ah, well, yes. I know what you are saying here. 

But I think the default value of keyword arguments is much simpler than `||=` idiom to do this same thing honestly😅 `client ||= Spaceship::ConnectAPI` appears everywhere right now so I just followed it for the sake of consistency but it should have been consistent with using "default value", IMHO. I may clean it up at some point 🤔 

https://github.com/fastlane/fastlane/pull/17475

(Moreover, I don't fully understand the point to pass a `client` class type to each method 🤔 )

Reply:
@ainame There are people who use these methods with multiple teams at the same time (which means multiple clients). These users are not able to just use the single session on the `Spaceship::ConnectAPI` for this reason. So these users create individual `Spaceship::ConnectAPI::Client` instances and pass them into the models to use. It's a very niche feature but something we need to support 🤷‍♂️

Reply:
@joshdholtz Yes, I know that but was just wondering what's the reason behind using multiple sessions🤔 I saw the PR changing it this way but it didn't contain much information, unfortunately.